sysvar/src/fees.rs

```diff
@@ -76,4 +76,52 @@ mod tests {
         let cloned_fees = fees.clone();
         assert_eq!(cloned_fees, fees);
     }
+
+    struct MockFeesSyscall;
+    impl crate::program_stubs::SyscallStubs for MockFeesSyscall {
+        fn sol_get_fees_sysvar(&self, var_addr: *mut u8) -> u64 {
+            let fees = Fees {
+                fee_calculator: FeeCalculator {
+                    lamports_per_signature: 42,
+                },
+            };
+            unsafe {
+                std::ptr::copy_nonoverlapping(
+                    &fees as *const _ as *const u8,
+                    var_addr,
+                    core::mem::size_of::<Fees>(),
+                );
+            }
+            solana_program_entrypoint::SUCCESS
+        }
+    }
+
+    #[test]
+    #[serial]
+    fn test_fees_get_deprecated_syscall_path() {
+        let prev = crate::program_stubs::set_syscall_stubs(Box::new(MockFeesSyscall));
+        let got = Fees::get().unwrap();
+        // restore original stubs to avoid side effects on other tests
+        let _ = crate::program_stubs::set_syscall_stubs(prev);
```

Comment:
Is this needed? It looks like all of the tests in this PR properly set the syscall stubs.

I'd consider it a bug if a test expects a particular set of stubs to be set without setting it itself.